Sources/Datadog/Logging/LoggingFeature.swift

```diff
@@ -72,19 +72,23 @@ internal final class LoggingFeature {
         return FeatureUpload(
             featureName: LoggingFeature.featureName,
             storage: storage,
-            uploadHTTPHeaders: HTTPHeaders(
+            uploadHTTPHeadersProvider: HTTPHeadersProvider(
                 headers: [
                     .contentTypeHeader(contentType: .applicationJSON),
                     .userAgentHeader(
                         appName: configuration.common.applicationName,
                         appVersion: configuration.common.applicationVersion,
                         device: commonDependencies.mobileDevice
-                    )
+                    ),
+                    .ddAPIKeyHeader(clientToken: configuration.clientToken),
+                    .ddEVPOriginHeader(source: configuration.common.source),
+                    .ddEVPOriginVersionHeader(),
+                    .ddRequestIDHeader(),
```

Comment:
There is [an open question](https://github.com/DataDog/dd-sdk-android/pull/671#discussion_r685768247) from @0xnm (in https://github.com/DataDog/dd-sdk-android/pull/671) if the `DD-REQUEST-ID` should be included all the time or only conditionally with a user setting.